packages/main/src/Input.js

```diff
@@ -581,17 +585,47 @@ class Input extends UI5Element {
 	}
 
 	get ariaDescribedBy() {
+		if (this._accInfo && this._accInfo.ariaDescribedBy) {
+			return `${this.suggestionsTextId} ${this.valueStateTextId} ${this._accInfo.ariaDescribedBy}`.trim();
+		}
 		return `${this.suggestionsTextId} ${this.valueStateTextId}`.trim();
 	}
 
 	get ariaHasPopup() {
+		if (this._accInfo && this._accInfo.ariaHasPopup) {
+			return this._accInfo.ariaHasPopup;
+		}
 		return this.showSuggestions ? "true" : undefined;
 	}
 
 	get ariaAutoComplete() {
+		if (this._accInfo && this._accInfo.ariaAutoComplete) {
+			return this._accInfo.ariaAutoComplete;
+		}
 		return this.showSuggestions ? "list" : undefined;
 	}
 
+	get roleAttribute() {
+		if (this._accInfo && this._accInfo.roleAttribute) {
```

Comment:
Should be fine to return undefined, try 
return this._accInfo && this._accInfo.roleAttribute;

Same for ariaOwns ariaExpanded